dataikuapi/dss/project.py

```diff
@@ -825,6 +825,54 @@ def new_job_definition_builder(self, job_type='NON_RECURSIVE_FORCED_BUILD'):
         warnings.warn("new_job_definition_builder is deprecated, please use new_job", DeprecationWarning)
         return JobDefinitionBuilder(self, job_type)
 
+    ########################################################
+    # Jupyter Notebooks
+    ########################################################
+    
+    def list_jupyter_notebooks(self, as_objects=True, active=False):
+        """
+        List the jupyter notebooks of a project.
+
+        :param bool as_objects: if True, return the jupyter notebooks as a :class:`dataikuapi.dss.notebook.DSSNotebook`
+                        notebook handles instead of raw JSON
+        :param bool active: if True, only return currently running jupyter notebooks.
+
+
+        :returns: The list of the notebooks - see as_objects for more information
+        :rtype: list
+        """
+        notebooks = self.client._perform_json("GET", "/projects/%s/jupyter-notebooks/" % self.project_key,
+                                         params={"active": active})
+        if as_objects:
+            return [DSSNotebook(self.client, notebook_state['projectKey'], notebook_state['name'], state=notebook_state) for notebook_state in notebooks]
+        else:
+            return notebooks
+
+    def get_jupyter_notebook(self, notebook_name):
+        """
+        Get a handle to interact with a specific jupyter notebook
+
+        :param str notebook_name: The name of the jupyter notebook to retrieve
+        :returns: A handle to interact with this jupyter notebook
+        :rtype: :class:`~dataikuapi.dss.notebook.DSSNotebook` jupyter notebook handle
+        """
+        notebook_content = self.client._perform_json("GET",
+                                             "/projects/%s/jupyter-notebooks/%s" % (self.project_key, notebook_name))
+        return DSSNotebook(self.client, self.project_key, notebook_name, content=notebook_content)
+
+    def create_jupyter_notebook(self, notebook_name, notebook_content):
+        """
+        Create a new jupyter notebook.
+
+        :param str notebook_name: the name of the notebook to create
+        :param dict notebook_content: the data of the notebook to create, as a dict.
+            The data will be converted to a JSON string internally.
+            Use ``get_state()`` on a similar existing ``DSSNotebook`` object in order to get a sample definition object
+        """
+        return self.client._perform_json("POST",
```

Comment:
You should return a handle to the newly create notebook and not the message.
I was playing with it and did:
`notebook = project.create_jupyter_notebook("MyName", { SUPER COMPLICATED DICT 😄  })`
Check for example:
`project.create_recipe()`

Moreover, the Public api returns directly a String /!\ It's a bad practice (I know we all doing that 😄 ) because in case you want to return more info, you will need to break the public API. Return an object that encapsulate the name of the notebook.

Reply:
The public API returns
```
{
    "msg": "Created Jupyter notebook 'nb4'"
}
```

Reply:
My bad 😄  I didn't pay too much attention to the WriteMessage method 🙈 . I still think that we should return a DSSNotebook from the method and not directly the message from the API. (We should pay attention to the save because the content will not be loaded)

Reply:
I feel weird about the message, after a creation, you should get a reference of the newly created object.
So, to be REST compliant, you should get something like:
```
{
    "id": "nb4",
    "msg": "Created Jupyter notebook 'nb4'"
}
```

Reply:
Yes I was looking at other api calls like recipe and they return the object on creation. Should we change it in a PR with the new sessions endpoint @apichery ?

Reply:
>Should we change it in a PR with the new sessions endpoint @apichery ?

Yes